src/pudl/constants.py

```diff
@@ -2097,6 +2105,15 @@
 
 ESTIMATED_OR_ACTUAL = {'E': 'Estimated', 'A': 'Actual'}
 
+TRANSIT_TYPE_DICT = {
+    'CV': 'Conveyer',
+    'PL': 'Pipeline',
+    'RR': 'Railroad',
+    'TK': 'Truck',
+    'WA': 'Water',
+    'UN': 'Unknown',
+}
```

Comment:
The same mapping is enumerated in a call to `df.replace()` in one of the transform functions. I think we should keep this information in just one place. If it's only used in the EIA 860 transform step, then we've been trying to move these types of constants into the modules where they're used rather than adding more stuff to `pudl.constants`  Also I had thought we were using an all lower-case convention for codes.

There's also the `pudl.constants.transport_modes_eia923` dictionary, which seems to have some overlap with these codes. Can they / should they be integrated into a single uniform thing?

Reply:
good question -- I'll look into how similar they are.

Reply:
RE: code convention, I thought it was upper case?

Reply:
Maybe we're talking about different things. Global variables like `TRANSIT_TYPE_DICT` are all caps. I was referring to values like `Conveyer` or `Railroad` -- is there any reason for them to be mixed case? I thought we were using `snake_case` -- it's nice to not have to ever have to remember capitalization, since it just opens up a bunch of different possible strings that all kind of sound like / feel like the same thing, but are not the same thing from the computer's point of view.

Reply:
ah that is a good point! I'll keep it to snake case